src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final double base;
+    private final double height;
+
+    public RightTriangle(Color color, double base, double height) {
+        super(color);
+        this.base = base;
+        this.height = height;
+    }
+
+    @Override
+    public void print() {
+        System.out.println("Figure: " + FigureName.RIGHT_TRIANGLE.name()
+                + ", area:" + getArea()
+                + ", sq.units, base:" + base
+                + ", units, height: " + height
+                + ", units, color: " + getColor());
```

Comment:
[To get a string representation of an enum use `name()`](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#use-name-for-getting-string-representation-of-enum-constants)